crates/circuit/src/dag_node.rs

```diff
@@ -114,8 +114,6 @@ impl DAGNode {
 #[pyclass(module = "qiskit._accelerate.circuit", extends=DAGNode)]
 pub struct DAGOpNode {
     pub instruction: CircuitInstruction,
```

Comment:
Nice! Maybe this should become a tuple struct now. Feel free to leave as is if that's your preference.

Reply:
I kept it this way for simplicity since I just deleted the lines with the previous attribute. I did overlook there is only a single field now. But I think I'd probably rather keep it the same because I feel like there are a few places we do `node.instruction`/`self.instruction` and that feels a bit more ergonomic than `node.0`